stuffioAPI/stuffioAPI/Models/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace stuffioAPI.Models
+{
+    public class User
+    {
+        public int Id { get; set; }
+
+        public string UserName { get; set; }
+
+        public string UserSurname { get; set; }
+
+        public string UserImg { get; set; }

```

Comment:
We don't pay for letters. Image may be written out.

Reply:
Got it. done